bokehjs/src/coffee/models/mappers/linear_mapper.coffee

```diff
@@ -10,23 +10,27 @@ class LinearMapper extends Model
     @add_dependencies('mapper_state', @get('source_range'), ['start', 'end'])
     @add_dependencies('mapper_state', @get('target_range'), ['start', 'end'])
 
+  @getters {
+    mapper_state: () -> @_get_computed('mapper_state')
```

Comment:
Anyway, at this point we can easily switch to using getters/setters across the code base with a few sed runs.